commercetools/symfony-bundle/0.4/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+    "bundles": {
+        "Commercetools\\Symfony\\CtpBundle\\CtpBundle": ["all"],
+        "Commercetools\\Symfony\\CustomerBundle\\CustomerBundle": ["all"],
+        "Commercetools\\Symfony\\SetupBundle\\SetupBundle": ["dev"],
+        "Commercetools\\Symfony\\StateBundle\\StateBundle": ["dev"]
```

Comment:
Are all these bundle living in this package?

Reply:
yes